frontend/poems/display-poems.js

```diff
@@ -14,18 +14,32 @@ function getPoems () {
   const req = { method: 'GET' }
   fetch('poems.json', req)
     .then(res => res.json())
-    .then(data => appendData(data))
+    .then(data => displayData(data))
     .catch(err => console.log('error: ' + err))
 }
```

Comment:
once u've pulled the latest changes from the class repo, u should be able to update this to:
```js
fetch('/api/poems', req)
```
the data u get back is going to be structured a little different from ur test file though, it will be an object which includes a property called `poemList` which is the array of poems itself. which means u're going to want to update the `then` callback to:
```js
    .then(data => displayData(data.poemList))
```
the way the poem objects themselves are structured is also different from ur test data, so u'll have to update ur `displayData` function as well (see my comment below)